src/Elmish.WPF/Binding.fs

```diff
@@ -209,24 +210,40 @@ module Binding =
       >> mapModel ValueOption.ofOption
       >> mapMsg ValueOption.toOption
 
+      
+  module Cmd =
 
-  module SubModel =
+    let createWithParam exec canExec autoRequery =
+      { Exec = exec
+        CanExec = canExec
+        AutoRequery = autoRequery }
+      |> CmdData
+      |> BaseBindingData
+      |> createBinding
+
+    let create exec canExec =
+      createWithParam
+        (fun _ -> exec)
+        (fun _ -> canExec)
+        false
+      >> addLazy (fun m1 m2 -> canExec m1 = canExec m2)
+
+  module OneWaySeq =
+    open BindingData.OneWaySeq
+
+    let internal create get itemEquals getId =
```

Comment:
I thought the idea of having files called `Binding.fs` and `BindingInternal.fs` is that the former only contains public things while the latter only contains non-public things.  I like this idea, but you are adding an internal function here.

Reply:
The split now has to do with whether it deals with `Binding` types or `BindingData` types.